<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmMultiLink</class>
 <widget class="QDialog" name="ConfirmMultiLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Link Identification Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../MultiLink.png</normaloff>../MultiLink.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayout_2">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>353</y>
     <width>178</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridlayAccept">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableInstallations">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>361</width>
     <height>386</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Site</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SiteID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lblTechnology">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>35</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Technology: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMinClear">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>175</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>0 - means Line of Sight; 60 means 60% of the 1st Fresnel Zone is clear; 100 means the entire 1st Fresnel Zone is clear.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxPath">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>115</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Path Loss (dB) : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTechnology">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>35</y>
     <width>156</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This is the technology from which the parameters will be derived and from which the radio installations will generated. The technology must have a Default Site.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinClear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>175</y>
     <width>136</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Clearance: </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbMaxPath">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>105</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxDist">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Distance (km) : </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbMaxDist">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblProject">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>76</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Project: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbProject">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>156</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The project to which the generated Radioinstallations should be assigned</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbKfactor">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>210</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>This is used to indicate the refractivity gradient in the lower troposphere</string>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblKfactor">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>k-Factor : </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>267</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save working links pdf files?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>315</y>
     <width>246</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPath">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>290</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Directory to save pdf files:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qrap.qrc"/>
 </resources>
 <connections/>
</ui>
